src/tabular-comparison.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
```

Comment:
This title isn't used right so it can be removed

Reply:
updated the title